server/manager/src/main/java/org/apache/accumulo/manager/tableOps/merge/MergeTablets.java

```diff
@@ -187,6 +187,20 @@ public Repo<Manager> call(FateId fateId, Manager manager) throws Exception {
             DeleteRows.getMergeTabletAvailability(range, tabletAvailabilities));
         tabletMutator.putPrevEndRow(firstTabletMeta.getPrevEndRow());
 
+        // TODO: How should we determine this value this after merging?
+        // Do we just keep the value that's already in the last tablet in the range?
```

Comment:
Keeping the one on the last tablet as is makes sense if we think of mergability being set on a split point.  So some mergabiity was set on the last split in the past, so need to keep it.  The other split points are being merged away, so no longer care about their mergability.